src/components/flat-table/flat-table-expandable.stories.mdx

```diff
@@ -0,0 +1,963 @@
+              <Icon type="pdf" />
+            </IconButton>
+          </BatchSelection>
+          <FlatTable>
+            <FlatTableHead>
+              <FlatTableRow>
+                <FlatTableCheckbox
+                  as="th"
+                  checked={selectAll}
+                  onChange={() => handleSelectAllRows()}
+                />
+                <FlatTableHeader>Name</FlatTableHeader>
+                <FlatTableHeader>Location</FlatTableHeader>
+                <FlatTableHeader>Relationship Status</FlatTableHeader>
+                <FlatTableHeader>Dependents</FlatTableHeader>
+              </FlatTableRow>
+            </FlatTableHead>
+            <FlatTableBody>
+              <FlatTableRow
+                onClick={() => handleHighlightRow("one")}
+                highlighted={highlightedRow === "one"}
+                expandable
+                subRows={subRows("one")}
+              >
+                <FlatTableCheckbox selectable={false} />
+                <FlatTableCell>John Doe</FlatTableCell>
+                <FlatTableCell>London</FlatTableCell>
+                <FlatTableCell>Single</FlatTableCell>
+                <FlatTableCell>0</FlatTableCell>
+              </FlatTableRow>
+              <FlatTableRow
+                onClick={() => handleHighlightRow("two")}
+                highlighted={highlightedRow === "two"}
+                expandable
+                subRows={subRows("two")}
+              >
+                <FlatTableCheckbox selectable={false} />
+                <FlatTableCell>Jane Doe</FlatTableCell>
+                <FlatTableCell>York</FlatTableCell>
+                <FlatTableCell>Married</FlatTableCell>
+                <FlatTableCell>2</FlatTableCell>
+              </FlatTableRow>
+              <FlatTableRow
+                onClick={() => handleHighlightRow("three")}
+                highlighted={highlightedRow === "three"}
+                expandable
+                subRows={subRows("three")}
+              >
+                <FlatTableCheckbox selectable={false} />
+                <FlatTableCell>John Smith</FlatTableCell>
+                <FlatTableCell>Edinburgh</FlatTableCell>
+                <FlatTableCell>Single</FlatTableCell>
+                <FlatTableCell>1</FlatTableCell>
+              </FlatTableRow>
+              <FlatTableRow
+                onClick={() => handleHighlightRow("four")}
+                highlighted={highlightedRow === "four"}
+                expandable
+                subRows={subRows("four")}
+              >
+                <FlatTableCheckbox selectable={false} />
+                <FlatTableCell>Jane Smith</FlatTableCell>
+                <FlatTableCell>Newcastle</FlatTableCell>
+                <FlatTableCell>Married</FlatTableCell>
+                <FlatTableCell>5</FlatTableCell>
+              </FlatTableRow>
+            </FlatTableBody>
+          </FlatTable>
+        </>
+      );
+    }}
+  </Story>
+</Preview>
```

Comment:
Think it might be a good idea to add the props table for FlatTableRow here, I know they are in the MDX but could be useful for quick checking as that is where the props for these stories are.